src/worldfactory.cpp

```diff
@@ -1385,6 +1290,12 @@ void worldfactory::draw_worldgen_tabs(WINDOW *w, unsigned int current)
         }
     }
 
+    static std::vector<std::string> tab_strings = {{
```

Comment:
Being `static` means the vector won't be re-translated when the language changes. This should probably be `const`, but not `static`.

Reply:
Or can be `static const`, but should not be translated here, instead just marked with `translate_marker` and translated later.